docs/content/getting-started/popular-articles.md

```diff
@@ -3,14 +3,11 @@ layout: popular-articles
 title: Popular Articles
 order: 3
 containerCls: container
-popularArticles:
-  - title: Additional properties
-    textPreview: You can define an additional property for a Granite UI widget or overwrite an existing one with the @Property annotation. This will be rendered into an appropriate subnode under the component's <cq:dialog> or <cq:design_dialog> node and will ...
-    url: /content/dev-tools/component-management/additional-properties
-  - title: Feeding data to selection widgets with OptionProvider
-    textPreview: Several Granite/Touch UI components, such as RadioGroup or Select, facilitate selecting from a set of options. Traditionally, the options are either inlined (the ToolKit offers its @RadioButton and @Option annotations for that) or supplied via a dat...
-    url: /content/dev-tools/option-provider
-  - title: Dynamic option change
-    textPreview: Because an @OptionProvider supports path references apart from regular paths, the setting that says "where to look for the path" can be stored in a dialog field other than the one that actually deals with paths.
-    url: /content/dev-tools/option-provider
+articles:
+    - /content/dev-tools/component-management/additional-properties
+    - /content/dev-tools/option-provider
+    - /content/dev-tools/component-management/component-structure
+    - title: Additional properties
+      content: You can define an additional property for a Granite UI widget or overwrite an existing one with the @Property annotation. This will be rendered into an appropriate subnode under the component's <cq:dialog> or <cq:design_dialog> node and will ...
```

Comment:
The possibility to override is requested but currently, there is no need to have it for the real content

Reply:
Yep